File: NewApp-SPA/src/app/_models/restaurant.ts
import { FoodItem } from './FoodItem';

export interface Restaurant {
<<<<<<< HEAD
  [x: string]: any;
=======
  
>>>>>>> d702adf5e127ff358fad8ba2750362ce1377c348
    id: number;
    name: string;
    address: string;
    contact_person: string;
    contact_number: number;
    email: string;
    website: string;
    timings: string;
    timeinterval: string;
    rating: number;
    type: string;
    ismain: boolean;
    url: string;
    fooditem: FoodItem[];

}

File: NewApp.API/Models/Restaurant.cs
using System.Collections.Generic;

namespace NewApp.API.Models
{
    public class Restaurant
    {
       public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }

        public string contact_person { get; set; }

        public int contact_number { get; set; }

        public string Email { get; set; }
        public string website { get; set; }
        
        public string Timings { get; set; }

        public string TimeInterval { get; set; }
        public int Rating { get; set; }
         public string Url { get; set; }
        public string type { get; set; }
        public ICollection<Fooditem> Fooditems { get; set; }
        public ICollection<Characteristic> Characteristics { get; set; }
        
    }
}